corehq/apps/hqwebapp/static/hqwebapp/yaml/requirejs.yaml

```diff
@@ -0,0 +1,56 @@
+mainConfigFile: corehq/apps/hqwebapp/static/hqwebapp/js/requirejs_config.js
+bundlesConfigOutFile: hqwebapp/js/requirejs_config.js
+baseUrl: staticfiles
+dir: staticfiles
+allowSourceOverwrites: true
+keepBuildDir: true
+skipDirOptimize: true   # could turn this off to minify everything
+fileExclusionRegExp: /(^\.)|(\.css$)|(CACHE)|(^formdesigner)/,
```

Comment:
Should `formdesigner` match the entire filename? i.e., `^formdesigner$`

Reply:
Did some testing and found the regex is only matching the file name, not the full path.  So the only  practical thing to exclude is css files, since I'm not doing css optimizations. Updated.